extensions/package-vulnerability-scanner/manifest.json

```diff
@@ -1,51 +1,49 @@
 {
-	"version": 1,
```

Comment:
Most of these are Prettier formatting changes. The only real changes are file checksums